pkg/txnutil/lock_resolver.go

```diff
@@ -131,7 +131,7 @@ func (r *resolver) Resolve(ctx context.Context, regionID uint64, maxVersion uint
 		}
 		bo = tikv.NewGcResolveLockMaxBackoffer(ctx)
 	}
-	log.Info("resolve lock successfully",
+	log.Debug("resolve lock successfully",
```

Comment:
I think this would be helpful to check if the region is locked, so I would prefer to keep it at the `Info` level.

Reply:
done